automation/test.sh

```diff
@@ -267,7 +267,7 @@ build_images() {
 export NAMESPACE="${NAMESPACE:-kubevirt}"
 
 # Make sure that the VM is properly shut down on exit
-trap '{ make cluster-down; }' EXIT SIGINT SIGTERM SIGSTOP
+trap '{ ret=$?; make cluster-down || true; exit $ret; }' EXIT SIGINT SIGTERM SIGSTOP
```

Comment:
@oshoval makes sense to me - however in theory make cluster-down might fail for a reason we might want to see, not shove under the carpet. While I think that is rather theory, I would want to have noticed if something like this happens. Would you mind looking at the causes why cluster-down has failed in recent times?

(will not block on this though)

Reply:
answered below
opened https://github.com/kubevirt/kubevirt/issues/13748